packages/core/src/shipping/create-shipping-strategy-registry.ts

```diff
@@ -30,6 +40,18 @@ export default function createShippingStrategyRegistry(
         new PaymentMethodRequestSender(requestSender),
     );
     const scriptLoader = getScriptLoader();
+    const subscriptionsActionCreator = new SubscriptionsActionCreator(
+        new SubscriptionsRequestSender(requestSender),
+    );
+    const billingAddressActionCreator = new BillingAddressActionCreator(
+        new BillingAddressRequestSender(requestSender),
+        subscriptionsActionCreator,
+    );
+    const braintreeHostWindow: BraintreeHostWindow = window;
+    const braintreeIntegrationService = new BraintreeIntegrationService(
+        new BraintreeScriptLoader(getScriptLoader(), braintreeHostWindow),
+        braintreeHostWindow,
+    );
```

Comment:
Could you please move this to line 88?

Reply:
I am not sure that keep so big construction as an argument it is a good idea. As for me it is less comfortable for reading. But if you think that it will be better, no problem, it moved.